### ISSUE TITLE
ArcGIS configuration: Add publication parameters

### ISSUE THREAD

**Issue by mboulstr1** [bcgov/whse_2_agol#8]

**Comment by bruce-tonkin** [comment]
First cut of the configuration files has been completed.  Need to verify the attributes work as expected and add a few more.

**Comment by bruce-tonkin** [comment]
Updated these 3 attributes in the config files for delivery only:
https://gogs.data.gov.bc.ca/waops/whse_2_agol/src/branch/master/app/mxd/config/service-config-dlvr.json
		"snippet": "Map Service Courtesy of DataBC",
		"access_information": "This service has been built for a specific purpose and may change at any time",
		"credits": "DataBC, Province of British Columbia"
Once the job has run we can find out what is missing.
So far I assume this is one that is outstanding:
https://delivery.maps.gov.bc.ca/arcgis/rest/services/whse/bcgw_pub_WHSE_LAND_AND_NATURAL_RESOURCE/MapServer
Need to update service description:
Service Description: DataBC bcgw_pub_WHSE_LAND_AND_NATURAL_RESOURCE
to
**Service Description: "This service has been built for a specific purpose and may change at any time"**
The other outstanding parameter is:
**Check and repair instances every: 	30	minutes**

**Comment by banders** [comment]
The principal task here is to ensure that the "snippet", "access_information", and "credits" fields are correctly transferred to the map services when published by mxd-publish.py.  Currently those fields aren't being correctly transferred.

**Comment by banders** [comment]
@bruce-tonkin I have updated the mxd publish process to set the "service description" and the "keep alive interval" from properties defined in the service-config file.  I just ran the latest version of the mxd publish process (in delivery).  All the services have been updated.  Here's an example:

https://delivery.maps.gov.bc.ca/arcgis/rest/services/whse/bcgw_pub_whse_legal_admin_boundaries/MapServer/info/metadata
